pages/Documents/ConversationalAI/ConversationBuilder/BotTemplates/order-status.md

```diff
@@ -16,9 +16,9 @@ The Order Status template is designed to provide users with information on retai
 The template uses text interactions only, so it can be deployed to any channel without modification. Escalation to an agent is also included.
 
 {: .important}
-This bot template contains a dialog template that can be used in other bots in your account. For more information on dialog templates, see [here](conversation-builder-dialog-templates.html).<br><br>This bot template also contains global functions brought over from the Global Helper Functions bot template. For information on these provided functions, see [here](conversation-builder-bot-templates-global-helper-functions.html).
+This bot template contains a [dialog template](conversation-builder-dialog-templates.html) that can be used in other bots in your account.<br><br>This bot template also contains global functions brought over from the [Global Helper Functions bot template](conversation-builder-bot-templates-global-helper-functions.html).
```

Comment:
Please use Markdown only where possible.

If this really needs two paragraphs, switch to HTML for the notice:

```
<div class="important">
<p>…</p>
<p>…</p>
</div>
```

But if possible, Markdown is strongly preferred.